Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -167,6 +167,12 @@ static bool nodeAndRendererAreValid(Node* node)
     return node ? nodeRendererIsValid(*node) : false;
 }
 
+static RefPtr<Element> getElementByIdWithReferenceTarget(const TreeScope& treeScope, const AtomString& elementId)
+{
+    RefPtr<Element> element = treeScope.getElementById(elementId);
```

Comment:
```suggestion
    RefPtr element = treeScope.getElementById(elementId);
```

WebKit style usually omits types

Reply:
Thanks, would you be able to help me understand the reasoning behind that?

Reply:
Because specifying `Element` here repeats the information already conveyed by the function name "getElementById".